Sources/Knit/Container+MainActor.swift

```diff
@@ -27,11 +25,12 @@ extension Container {
     public func register<Service>(
         _ serviceType: Service.Type,
         name: String? = nil,
-        mainActorFactory: @escaping @MainActor (Resolver) -> Service
+        factory: @escaping @MainActor (TargetResolver) -> Service
```

Comment:
Shouldn't these parameter names stay the same? Knit uses it to determine whether a factory should be on the main actor

Reply:
Yeah my mistake on this one, updating